src/build-config/mageos-nightly-build-config.js

```diff
@@ -0,0 +1,61 @@
+
+const packagesConfig = require('./packages-config');
+
+const branchBuildConfig = {
+  'magento2': {
+    repoUrl: 'https://github.com/mage-os/mageos-magento2.git',
+    ref: '2.4-develop',
+  },
+  'security-package': {
+    repoUrl: 'https://github.com/mage-os/mageos-security-package.git',
+    ref: 'develop',
+  },
+  'inventory': {
+    repoUrl: 'https://github.com/mage-os/mageos-inventory.git',
+    ref: 'develop',
+  },
+  'inventory-composer-installer': {
+    repoUrl: 'https://github.com/mage-os/mageos-inventory-composer-installer.git',
+    ref: 'master',
+  },
+  'page-builder': {
+    repoUrl: 'https://github.com/mage-os/mageos-magento2-page-builder.git',
+    ref: 'develop',
+  },
+  // Starting from Adobe Commerce and Magento Open Source 2.4.5 release Adobe IMS Integration package became part of Magento Open Source project.
```

Comment:
Why is this commented out?

Reply:
Because it is present in the mirror build config. This way it is clear that it is not included on purpose.